pkg/sloop/webserver/webserver.go

```diff
@@ -66,23 +65,27 @@ type WebConfig struct {
 }
 
 var (
-	metricWebServerRequestCount   = promauto.NewCounter(prometheus.CounterOpts{Name: "sloop_webserver_request_count"})
-	metricWebServerRequestLatency = promauto.NewGauge(prometheus.GaugeOpts{Name: "sloop_webserver_request_latency_sec"})
+	metricWebServerRequestCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "sloop_webserver_http_requests_total",
+			Help: "A counter for requests to the wrapped handler.",
+		},
+		[]string{"code", "handler"},
+	)
+	metricWebServerRequestDuration = promauto.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "sloop_webserver_http_request_duration_seconds",
+			Help:    "A histogram of latencies for requests to the wrapped handler.",
+			Buckets: []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 30, 60},
```

Comment:
Potentially the number of buckets could be reduced for now as we are querying a presumably local BadgerDB.
If the data store is changed in the future then it may be possible to see longer durations.
I could go either way.